config/initializers/job_configurations.rb

```diff
@@ -209,7 +210,7 @@
       # Queue usps proofing job to GoodJob
       get_usps_proofing_results_job: {
         class: 'GetUspsProofingResultsJob',
-        cron: cron_1h,
+        cron: cron_10m,
```

Comment:
Let's also consider making this a config value so that if 10m still creates too much load, we can update it without having to deploy a code change

Reply:
created LG-6980 for this